waves/src/App.tsx

```diff
@@ -250,40 +228,53 @@ function App() {
 
     let walletBalances;
 
-    if (!walletStatus.exists) {
-        walletBalances = <Button
-            onClick={onCreateWalletOpen}
-            size="sm"
+    async function unlock_wallet() {
+        // TODO send request to open popup to unlock wallet
+        debug("For now open popup manually...");
+        await reloadWalletStatus;
+    }
+
+    async function get_extension() {
+        // TODO forward to firefox app store
```

Comment:
TODO: Forward to browser-specific app store

Reply:
We only support firefox right now ;)